packages/react-composites/src/composites/common/AddPeopleButton.tsx

```diff
@@ -17,18 +17,29 @@ import {
 } from './styles/PeoplePaneContent.styles';
 /* @conditional-compile-remove(PSTN-calls) */
 import { AddPeopleDropdown } from './AddPeopleDropdown';
+/* @conditional-compile-remove(PSTN-calls) */
+import { CommunicationIdentifier } from '@azure/communication-common';
+/* @conditional-compile-remove(PSTN-calls) */
+import { AddPhoneNumberOptions } from '@azure/communication-calling';
 
 /** @private */
 export interface AddPeopleButtonProps {
   inviteLink?: string;
   mobileView?: boolean;
   participantList?: JSX.Element;
   strings: CallWithChatCompositeStrings | /* @conditional-compile-remove(one-to-n-calling) */ CallCompositeStrings;
+  /* @conditional-compile-remove(PSTN-calls) */
+  onAddParticipant: (participant: CommunicationIdentifier, options?: AddPhoneNumberOptions) => void;
+  alternateCallerId?: string;
 }
 
 /** @private */
 export const AddPeopleButton = (props: AddPeopleButtonProps): JSX.Element => {
   const { inviteLink, mobileView, strings, participantList } = props;
+  /* @conditional-compile-remove(PSTN-calls) */
+  const { alternateCallerId } = props;
```

Comment:
we can just use props.alternateCalledId directly below to remove these conditional statements